mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/GraphicsPathIteratorTest.cs

```diff
@@ -73,85 +73,79 @@ public void NextSubpath_Null ()
 		}
 
 		[Test]
-		[ExpectedException (typeof (NullReferenceException))]
 		public void CopyData_NullPoints ()
 		{
 			using (GraphicsPath gp = new GraphicsPath ()) {
 				gp.AddLines (pts_2f);
 				using (GraphicsPathIterator gpi = new GraphicsPathIterator (gp)) {
 					PointF [] points = null;
 					byte [] types = new byte [1];
-					Assert.AreEqual (0, gpi.CopyData (ref points, ref types, 0, 1));
+					Assert.Throws<NullReferenceException> (() => Assert.AreEqual (0, gpi.CopyData (ref points, ref types, 0, 1)));
```

Comment:
This looks very odd, it should either throw NRE or verify the result is 0. It cannot do both at the same time